src/PrestaShopBundle/Form/Admin/Configure/AdvancedParameters/Administration/FormDataProvider.php

```diff
@@ -35,10 +35,6 @@
  */
 final class FormDataProvider implements FormDataProviderInterface
 {
-    public const ERROR_NOT_NUMERIC_OR_LOWER_THAN_ZERO = 1;
```

Comment:
are these constants already part of a release ? if yes that is a BC Break

Reply:
You are right, it's BC break will add them back and depricate